src/natcap/invest/urban_mental_health.py

```diff
@@ -0,0 +1,1335 @@
+"""InVEST Urban Mental Health model."""
+import logging
+import os
+import shutil
+import tempfile
+
+import numpy
+import pandas
+import pygeoprocessing
+import pygeoprocessing.kernels
+from osgeo import gdal
+from osgeo import ogr, osr
+
+from . import gettext
+from . import spec
+from . import validation
+from .unit_registry import u
+
+LOGGER = logging.getLogger(__name__)
+
+SCENARIO_OPTIONS = [
+    spec.Option(key="tcc_ndvi", display_name="TCC & Baseline NDVI"),
+    spec.Option(key="lulc", display_name="Baseline & Alternate LULC"),
+    spec.Option(key="ndvi", display_name="Baseline & Alternate NDVI")
+    ]
+
+MODEL_SPEC = spec.ModelSpec(
+    model_id="urban_mental_health",
+    model_title=gettext("Urban Mental Health"),
+    userguide="",  # TODO - add this model to UG
+    validate_spatial_overlap=True,
+    different_projections_ok=True,
+    aliases=("umh",),
+    module_name=__name__,
+    input_field_order=[
+        ["workspace_dir", "results_suffix"],
+        ["aoi_vector_path", "population_raster", "search_radius"],
+        ["effect_size", "baseline_prevalence_vector",
+         "health_cost_rate"],
+        ["scenario"],
+        ["ndvi_base", "ndvi_alt"],
+        ["lulc_base", "lulc_alt", "lulc_attr_csv"],
+        ["tc_raster", "tc_target"]
+    ],
+    inputs=[
+        spec.WORKSPACE,
+        spec.SUFFIX,
+        spec.N_WORKERS,
+        spec.AOI.model_copy(update=dict(  #  TODO: potentially want to have this req to be census tract pop. shp? or only req that if opt 1 but optional for opts 2-3?
+            id="aoi_vector_path",
+            about=gettext(
+                "Map of the area over which to run the model. The AOI must be "
+                "smaller than the raster inputs by at least the search radius "
+                "to ensure correct edge pixel calculation, as InVEST will "
+                "buffer your AOI by the `search_radius` to determine the "
+                "processing area. Final outputs will be clipped to this AOI."),
+            projected=True,
+            projection_units=u.meter
+        )),
+        spec.SingleBandRasterInput(
+            id="population_raster",
+            name=gettext("population"),
+            about=gettext(
+                "Gridded population data representing the number of people "
+                "per pixel, ideally at the pixel level. Where available, "
+                "additional demographic attributes (e.g., age, income) can "
+                "enhance model accuracy and equity assessments. For "
```

Comment:
I'm confused by the "additional..." statement. Is this implying some population rasters could reflect certain demographic population count and that it would be a good use in this model?